modules/images/webp-uploads/load.php

```diff
@@ -219,28 +272,30 @@ function webp_uploads_generate_image_size( $attachment_id, $size, $mime ) {
  */
 function webp_uploads_get_supported_image_mime_transforms() {
```

Comment:
Since we're changing the filter name, let's change the function name accordingly as well. For example to `webp_uploads_get_upload_image_mime_transforms()`.